src/protocolsupport/protocol/packet/middleimpl/writeable/play/v_pe/PlayerListItemPacket.java

```diff
@@ -0,0 +1,44 @@
+package protocolsupport.protocol.packet.middleimpl.writeable.play.v_pe;
+
+import io.netty.buffer.ByteBuf;
+import net.md_5.bungee.protocol.packet.PlayerListItem;
+import protocolsupport.protocol.packet.middle.WriteableMiddlePacket;
+import protocolsupport.protocol.serializer.MiscSerializer;
+import protocolsupport.protocol.serializer.VarNumberSerializer;
+import protocolsupport.utils.netty.Allocator;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+
+public class PlayerListItemPacket extends WriteableMiddlePacket<PlayerListItem> {
+
+    @Override
+    public Collection<ByteBuf> toData(PlayerListItem origin) {
+        ByteBuf buf = cache.tabList().remove(origin);
+        if (!(buf == null)) {
+            return Collections.singletonList(buf);
+        }
+        switch (origin.getAction()) {
+            case ADD_PLAYER:
+            case UPDATE_GAMEMODE:
+            case UPDATE_LATENCY:
+            case UPDATE_DISPLAY_NAME:
+                return Collections.emptyList();
```

Comment:
Some bungee plugins send their own tablist packets, so just returning nothing here won't work.

Reply:
I think compatible with custom tablist plugin is not a good solution. pe's player list is much different with pc's tablist

Reply:
I means the visual different. Plugins put lots of extra infos into tablist but pe's playerlist is better to only handle player name.